system-tests/test/vite_dev_server_fresh_spec.ts

```diff
@@ -0,0 +1,25 @@
+import systemTests from '../lib/system-tests'
+
+describe('@cypress/vite-dev-server-fresh', function () {
+  systemTests.setup()
+
+  describe('react', () => {
+    systemTests.it('runs specs in vite 2.8.6', {
+      project: 'vite2.8.6-react',
```

Comment:
why aren't we setting `snapshot: true` here? is it because of non-deterministic `stdout` coming from the vite bundler? could we pass an environment variable or an `options: { ... }` into vite to silence this or log it to `stderr` ?

Reply:
Added snapshots